test/coins-test.js

```diff
@@ -48,7 +48,7 @@ describe('Coins', function() {
     assert.strictEqual(entry.height, 1);
     assert.strictEqual(entry.coinbase, false);
     assert.strictEqual(entry.raw, null);
-    assert.instanceOf(entry.output, Output);
+    assert(entry.output instanceof Output);
```

Comment:
Interesting, should we add this to `bsert` ?

Reply:
I don't think so, it's simple enough as-is?